### PR TITLE
Add RickDBCN email to plugins

### PR DESCRIPTION
**Pull request by RickDBCN**

### CODE REVIEW

**Comment by pxlrbt** [comment]
Is this really a widget and action? I think `panel-builder` is the better category. Or does the action can be used on its own?

**Comment by RickDBCN** [comment]
> Is this really a widget and action? I think `panel-builder` is the better category. Or does the action can be used on its own?

You are correct. Fixed!

**Review by sacalito — APPROVED** [review]

**Comment by sacalito** [comment]
sorry clicked by error, please remove the approved changes and this coment

File: content/plugins/rickdbcn-email [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Email Log Lite
+slug: rickdbcn-email
+author_slug: rickdbcn
+categories: [panel-builder, analytics]
+description: Log all outgoing emails in your project within your Filament panel.
+discord_url:
+docs_url: https://github.com/RickDBCN/filament-email/blob/main/README.md
```
Comment (danharrin) [inline content/plugins/rickdbcn-email:8]: This needs to be the raw markdown file path, like in our README example

**Review by danharrin — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.

**Pull request merged**